<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QMainWindow" name="Console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入人数[2,16]：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="node_num">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="node_0">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_10">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_11">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_12">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_13">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_14">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="node_15">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_confirm">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_reset">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gen_rand">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>摇骰子</string>
    </property>
   </widget>
   <widget class="QLabel" name="rand_num">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="out">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>已出列：</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="out_0">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1 </string>
    </property>
   </widget>
   <widget class="QLabel" name="out_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_10">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_11">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_12">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_13">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_14">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_15">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QLabel" name="tip">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>点击选择何处开始</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
